### PR TITLE
 DP-3625-Create-example-code 

### PR DESCRIPTION
**Pull request by Bryan-Noel**

### CODE REVIEW

File: src/main/java/com/northconcepts/datapipeline/examples/shopify/ReadShopifyInventoryItemsWriteToJson.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+import com.northconcepts.datapipeline.foundations.file.LocalFileSink;
+import com.northconcepts.datapipeline.foundations.pipeline.Pipeline;
+import com.northconcepts.datapipeline.foundations.pipeline.PipelineInput;
+import com.northconcepts.datapipeline.foundations.pipeline.PipelineOutput;
+import com.northconcepts.datapipeline.foundations.pipeline.input.DataReaderPipelineInput;
+import com.northconcepts.datapipeline.foundations.pipeline.output.JsonRecordPipelineOutput;
+import com.northconcepts.datapipeline.shopify.ShopifyInventoryItemReader;
+
+public class ReadShopifyInventoryItemsWriteToJson {
```
Comment (DeleTaylor) [inline src/main/java/com/northconcepts/datapipeline/examples/shopify/ReadShopifyInventoryItemsWriteToJson.java]: No need to say everything in the class name: `ReadShopifyInventoryItems` is enough.  We'll create a too later that follows your thinking.

Please update the other examples similarly.

File: src/main/java/com/northconcepts/datapipeline/examples/shopify/ReadShopifyInventoryItemsWriteToJson.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+                .setIds("49129939697963", "49129939730731", "49129939763499", "49129939796267", "49129939829035"));
+
+        PipelineOutput pipelineOutput = new JsonRecordPipelineOutput().setPretty(true)
+            .setFileSink(new LocalFileSink().setPath("data/output/output.json"));
+
+        Pipeline pipeline = new Pipeline()
+            .setInput(pipelineInput)
+            .setOutput(pipelineOutput);
+
+        pipeline.run();
```
Comment (DeleTaylor) [inline src/main/java/com/northconcepts/datapipeline/examples/shopify/ReadShopifyInventoryItemsWriteToJson.java]: Let's create a simple DataReader, DataWriter, & Job.run(). code -- no need for DPF here.

Please update the other examples similarly.

File: src/main/java/com/northconcepts/datapipeline/examples/shopify/ReadShopifyOrdersWriteToCsv.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+import com.northconcepts.datapipeline.shopify.client.ShopifyOrder;
+
+import java.io.File;
+
+public class ReadShopifyOrdersWriteToCsv {
+
+    private static final String DOMAIN = "https://your-store-domain.com";
+    private static final String TOKEN = "store-token";
+    
+    public static void main(String[] args) {
```
Comment (DeleTaylor) [inline src/main/java/com/northconcepts/datapipeline/examples/shopify/ReadShopifyOrdersWriteToCsv.java]: The code looks good -- very simple, just the essentials 👍

**Review by DeleTaylor — CHANGES_REQUESTED** [review]

File: src/main/java/com/northconcepts/datapipeline/examples/shopify/ReadShopifyInventoryItemsWriteToJson.java (hunk 1 above)
Comment (Bryan-Noel) [reply src/main/java/com/northconcepts/datapipeline/examples/shopify/ReadShopifyInventoryItemsWriteToJson.java]: done

File: src/main/java/com/northconcepts/datapipeline/examples/shopify/ReadShopifyInventoryItemsWriteToJson.java (hunk 2 above)
Comment (Bryan-Noel) [reply src/main/java/com/northconcepts/datapipeline/examples/shopify/ReadShopifyInventoryItemsWriteToJson.java]: done

**Review by DeleTaylor — APPROVED** [review]

**Pull request merged**